documentation/src/main/asciidoc/topics/ref_rolling_upgrade_remote_store.adoc

```diff
@@ -0,0 +1,32 @@
+[id='upgrade_remote_store']
+= Remote Cache Stores for Rolling Upgrades
+You must use specific remote cache store configuration to perform rolling
+upgrades, as follows:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<persistence passivation="false"> <1>
+   <remote-store xmlns="urn:infinispan:config:store:remote:{schemaversion}"
+                 cache="myDistCache" <2>
+                 hotrod-wrapping="true" <3>
+                 shared="true" <4>
+                 marshaller="" <5>
```

Comment:
@ryanemerson for remote cache stores with source clusters for older versions, e.g. remote store that points to a 9.4 cluster, do users need to add the jboss marshaller to the remote store config?

Reply:
nope. I tested migration to RHDG 8 from 7.3, 7.2, ..., 6.6.2 using this config:
```json
{
    "distributed-cache":{
        "mode":"SYNC",
        "persistence":{
            "remote-store":{
                "protocol-version":"2.5",
                "hotrod-wrapping":true,
                "raw-values":true,
                "cache":"default",
                "remote-server":{
                    "host":"127.0.0.1",
                    "port":11222
                }
            }
        }
    }
}
```

Reply:
https://github.com/gustavonalle/rolling/